client/cs/src/StateAccessors.cs

```diff
@@ -43,17 +82,17 @@ public States.State_F32 State(States.StateHandle_F32 state_handle)
             return new States.State_F32(p, OwnershipSemantics.SharedRef);
         }
 
-        public States.State_U8 StateMut(States.StateHandle_U8 state_handle)
+        public States.State_F32 StateMut(States.StateHandle_F32 state_handle)
```

Comment:
Reorganized the class so that the `State` and `StateMut` functions for a particular type are adjacent.